client/src/Components/QuoteCard/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image, Typography } from 'antd';
+import './style.css';
+
+const { Paragraph, Title } = Typography;
+
+function QuoteCard({ quote: { name, desc, img } }) {
+  return (
+    <div className="quote-card">
+      <Image
+        preview={false}
+        width={120}
+        height={120}
+        src={img}
+        alt="person-img"
```

Comment:
Keep in mind that this will be read by screen reader and this migh be read `person dash I M G` which is wrong
```suggestion
        alt="person image"
```